docs/config/README.md

```diff
@@ -2204,7 +2204,7 @@ By default the module will be shown if any of the following conditions are met:
 | ------------------- | ------------------------------------ | ----------------------------------------------------------------------------------------------------- |
 | `format`            | `"via [$symbol($version )]($style)"` | The format for the module.                                                                            |
 | `version_format`    | `"v${raw}"`                          | The version format. Available vars are `raw`, `major`, `minor`, & `patch`                             |
-| `symbol`            | `" "`                               | A format string representing the symbol of Node.js.                                                   |
+| `symbol`            | `"⬢ "`                               | A format string representing the symbol of Node.js.                                                   |
```

Comment:
This symbol is already taken by the container module: https://github.com/starship/starship/blame/98afd217c6002feb007b3c21c16ea22c22f7374d/docs/config/README.md#L686

But maybe that one could be replaced with 🔴  :-)

Reply:
Others, please respond with your opinions as well before I make any change (To save me from shuffling things back and forth 😝).

Reply:
We do already have duplicate symbols in starship (e.g. .NET's TFM and Dart use the same symbol, while perl and ocaml render identically in some emoji sets) so this isn't a killer. I'd like to get reviews from other maintainers signing off on this idea before proceeding though, since this has traditionally been a contentious issue.

Reply:
Also, note that container is default `bold red dimmed` while nodejs is `bold green` so I don't think there is a risk these will be mistaken except by our colorblind users. It also depends on the colorscheme being used. I'll take a look with MacOS's colorblind filters at some point to see if this will be an issue under popular colorschemes.

Reply:
Note: this is what it looks like in my system under red-green colorblindness simulation.

<img width="224" alt="image" src="https://user-images.githubusercontent.com/4605384/155792798-3fdd09a4-0d26-4215-8df8-8a1e51a1d086.png">

While it's dependent on colorscheme, I tried all default colorschemes in iTerm2 and they are easily distinguishable side-by-side (though under one or two schemes, they might be difficult to distinguish separately).